src/Altinn.AccessManagement.Core/Constants/AuthzConstants.cs

```diff
@@ -19,5 +19,15 @@ public static class AuthzConstants
         /// Policy tag for authorizing internal Altinn.Platform.Authorization API access
         /// </summary>
         public const string INTERNAL_AUTHORIZATION = "InternalAuthorizationAccess";
+
+        /// <summary>
+        /// Policy tag for reading an instance
+        /// </summary>
+        public const string POLICY_MASKINPORTEN_DELEGATION_READ = "InstanceRead";
```

Comment:
Could we rename this from instanceread? Since it is "taken" by apps.  MaskinportenDelegationRead or something shorter